### PR TITLE
Add --noconsole to suppress console output when creating a one-off VM

### PR DESCRIPTION
**Pull request by zdykstra**
The output, especially in tmux, is pretty jumbled and messy. I'd prefer to have it disabled by default, but I don't know the general use-cases people have for `start`. I left the current default behavior alone.

### CODE REVIEW

File: virt_lightning/shell.py [hunk 1]
```diff
@@ -143,21 +143,26 @@ def start(configuration, context, **kwargs):
     domain = _start_domain(hv, host, context, configuration)
     if not domain:
         return
-    import time
 
-    time.sleep(4)
-    stream = conn.newStream(libvirt.VIR_STREAM_NONBLOCK)
-    console = domain.dom.openConsole(None, stream, 0)
     loop = asyncio.get_event_loop()
```
Comment (zloidemon) [inline virt_lightning/shell.py]: I think this line should placed below `import libvirtaio`, no reasons init loop without using.

Comment (zdykstra) [reply virt_lightning/shell.py]: There's always a consumer of `loop`, it's used to async await for `domain.reachable()`.

Comment (zdykstra) [reply virt_lightning/shell.py]: https://github.com/zdykstra/virt-lightning/blob/suppress/virt_lightning/shell.py#L167-L170

Comment (zloidemon) [reply virt_lightning/shell.py]: Ah, sorry didn't see in diff. This patch is fine

**Comment by zdykstra** [comment]
Ahh yeah, the diff was a bit deceptive!

**Pull request merged**